project1/css/index.css

```diff
@@ -54,11 +54,92 @@ table {
 
 html, body {
     height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+	font-family: 'Titillium Web', sans-serif;
 }
 
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+body{
+	display: flex;
+	justify-content: center;
+}
+.container {
+  width: 800px;
+}
+
+header {
+  display: flex;
+  width: 100%;
+  
```

Comment:
You can add `align-items: center;` here so your `nav` sits nicely centered inside `header`